doc/docportal/other_platforms/ios.rst

```diff
@@ -47,11 +47,11 @@ Create a new directory called ``build`` at the same level as the repository you
 
     mkdir build
 
-Copy the contents of the ``scummvm-ios7-libs-v2`` folder you extracted earlier into the ``build`` directory, using either the Finder, or the command line as follows:
+Copy the contents of the ``scummvm-ios7-libs-v3`` folder you extracted earlier into the ``build`` directory, using either the Finder, or the command line as follows:
 
 .. code-block::
 
-    cp -r ~/Downloads/scummvm-ios7-libs-v2/* ~/build/
+    cp -r ~/Downloads/scummvm-ios7-libs-v3/* ~/build/
```

Comment:
The new package is extracted as `frameworks` for me and not `scummvm-ios7-libs-v3/frameworks`.
So the instruction becomes: `cp -r ~/Downloads/frameworks ~/build/`
Or maybe we should suggest `mv ~/Downloads/frameworks ~/build/`